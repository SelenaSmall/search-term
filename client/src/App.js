import React, { Component } from 'react';
import Game from './components/Game'
import Routes from './Routes'
import { BrowserRouter } from 'react-router-dom'

class App extends Component {
    constructor(props) {
        super(props);
    }

    render() {
        return (
            <BrowserRouter>
                <Routes />
            </BrowserRouter>
    );
    }
}

export default App;