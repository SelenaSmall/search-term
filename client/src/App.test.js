import React from 'react';
import ReactDOM from 'react-dom';
import Enzyme, { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import Adapter from 'enzyme-adapter-react-16';
import App from './App';

Enzyme.configure({ adapter: new Adapter() });

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<App />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('snapshot testing', () => {
  const output = shallow(<App />);
  expect(shallowToJson(output)).toMatchSnapshot();
});
