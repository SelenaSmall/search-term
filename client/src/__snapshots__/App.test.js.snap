// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot testing 1`] = `
<div>
  <BrowserRouter>
    <_default />
  </BrowserRouter>
  <_default />
</div>
`;
